circle.yml

```diff
@@ -4,8 +4,8 @@ jobs:
     # Build Cyclus
     build:
         docker:
-            - image: cyclus/cyclus:latest
-        working_directory: ~/recyle
```

Comment:
I don't think this should not depend on `cycamore`....
maybe you can use the sink/source that are provided by `cyclus` ?